Core/include/Acts/Detector/ProtoBinning.hpp

```diff
@@ -0,0 +1,94 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Common.hpp"
+#include "Acts/Utilities/BinningData.hpp"
+#include "Acts/Utilities/detail/AxisFwd.hpp"
+
+#include <stdexcept>
+#include <vector>
+
+namespace Acts {
+
+namespace Experimental {
+
+/// @brief  Simple helper class to define a binning structure
+///
+/// @note no checks are performed on the consistency, this is
+/// only for convenience that the binning can be defined and then
+/// translated into concrete axis types
+struct ProtoBinning {
```

Comment:
Allows to define a binning (even with job options), and then this is translated.